Tests/DiligentToolsTest/src/RenderStateNotationParser/RenderStateNotationParserTest.cpp

```diff
@@ -151,6 +151,82 @@ TEST(Tools_RenderStateNotationParser, TilePipelineNotationTest)
     ASSERT_EQ(*pDesc, DescReference);
 }
 
+TEST(Tools_RenderStateNotationParser, InlinePipelineStatesTest)
+{
+    RefCntAutoPtr<IRenderStateNotationParser> pParser = LoadFromFile("InlinePipelineStates.json");
+    ASSERT_NE(pParser, nullptr);
+
+    {
+        auto pDesc = pParser->GetGraphicsPipelineStateByName("Graphics-TestName");
+        ASSERT_NE(pDesc, nullptr);
+
+        auto pVSShader = pParser->GetShaderByName(pDesc->pVSName);
+        ASSERT_NE(pVSShader, nullptr);
```

Comment:
Please verify the correctness of values that are set in the JSON, not just that the object is not null as it may be incorrect (but not null).